src/Bundle/AutoMapperBundle/DependencyInjection/Configuration.php

```diff
@@ -29,6 +29,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->scalarNode('cache_dir')->defaultValue('%kernel.cache_dir%/automapper')->end()
                 ->scalarNode('date_time_format')->defaultValue(\DateTimeInterface::RFC3339)->end()
                 ->booleanNode('hot_reload')->defaultValue($this->debug)->end()
+                ->booleanNode('map_private_properties')->defaultTrue()->end()
```

Comment:
Since it's default to `false` in Symfony, I would prefer to follow their default when making a bundle.

I think we should do the same for the static `create` method for clarity.

Reply:
:+1: